src/components/grid/GridComponent.vue

```diff
@@ -128,31 +152,72 @@ export default Vue.extend({
     rsql: function () {
       this.loadGridData()
     }
-  },
-  created () {
-    this.loadAssessments()
-    this.loadGridData()
   }
 })
 </script>
 
 <style scoped>
+  table {
+    overflow: hidden;
+    position: relative;
+  }
+  table th:first-child {
+    width: 15rem;
+    max-width: 15rem;
+    overflow: hidden;
+  }
+  table td,
+  table th:not(:first-child){
+    width: 4rem;
+    max-width: 4rem;
+    min-width: 4rem;
+  }
   table th {
     white-space: nowrap;
     vertical-align: middle;
     font-weight: normal;
   }
+  .sticky{
+    position: fixed;
+    top:0;
+    background-color: white;
+    background: linear-gradient(180deg, rgba(255,255,255,1) 0%, rgba(255,255,255,0.9) 75%, rgba(255,255,255,0) 100%);
+    z-index: 2000;
+  }
+  .sticky .assessments-title{
+    height: 8rem;
+  }
+  .sticky .assessments-title span{
+    bottom: 1rem;
+  }
+  .space-holder{
+    height: 96px;
+  }
 
   table td, th {
     padding: 0 1px;
+    position: relative;
   }
 
-  .variable-title {
-    max-width: 12rem;
+  .col-hover td:hover::after {
+    content:"";
+    left: 0;
+    right: 0;
     display: inline-block;
+    position: absolute;
+    background-color: #e9f6f9;
+    top: -5000px;
+    height: 10000px;
+    z-index: -1;
```

Comment:
why -z , what z schema are we using here

Reply:
-z puts object under normal rendering depth of 0 needed to get the pseudo element under the parent element.